pandas/core/generic.py

```diff
@@ -3325,30 +3330,35 @@ def to_latex(
             tabular output.
         DataFrame.to_html : Render a DataFrame as an HTML table.
 
+        Notes
+        -----
+
+        .. note::
+           As of v2.0.0 this method has changed to use the Styler implementation as
+           part of :meth:`.Styler.to_latex` via ``jinja2`` templating. It is advised
```

Comment:
Might be good to explicitly mention that `jinij2` now needs to be installed for this to work?

Reply:
I think Styler returns something similar if you try to call it without jinja2 installed, but can also add a note here